var/spack/repos/builtin/packages/py-ipyrad/package.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyIpyrad(PythonPackage):
+    """An interactive toolkit for assembly and analysis of restriction-site
+    associated genomic data sets (e.g., RAD, ddRAD, GBS) for population
+    genetic and phylogenetic studies."""
+
+    homepage = "https://github.com/dereneaton/ipyrad"
+
+    url = "https://github.com/dereneaton/ipyrad/archive/refs/tags/0.9.85.tar.gz"
+
+    version("0.9.90", sha256="8b95aa3bae30da15baba90abb03176932411ff708c54d5e4481b811cceb8a4a8")
+    version("0.9.85", sha256="17b07466531655db878919e426743ac649cfab2e92c06c4e45f76ee1517633f9")
+
+    depends_on("python@3:", type=("build", "run"))
+    depends_on("py-pip", type="build")
+    depends_on("py-wheel", type="build")
+    depends_on("py-setuptools", type="build")
+
+    depends_on("bedtools2", type=("build", "run"))
+    depends_on("bwa", type=("build", "run"))
+    depends_on("muscle", type=("build", "run"))
+    depends_on("py-notebook", type=("build", "run"))
+    depends_on("samtools", type=("build", "run"))
+    depends_on("vsearch", type=("build", "run"))
```

Comment:
Might be useful to add a comment pointing to the docs where these deps were found. I only check `setup.py`